config/webpack.dev.js

```diff
@@ -22,6 +22,7 @@ const PORT = process.env.PORT || 3000;
 const HMR = helpers.hasProcessFlag('hot');
 // if env is 'inmemory', the inmemory debug resource is used
 const API_URL = process.env.API_URL || (ENV==='inmemory'?'app/':'http://localhost:8080/api/');
+const STACK_API_URL = process.env.STACK_API_URL || (ENV==='inmemory'?'app/':'http://localhost:8080/api/');
```

Comment:
This should not just copy the line above. You should set it to the correct value. And I really don't think you need the conditional checking if it's using 'inmemory'.